Index: .idea/.name
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>Hello Zupper
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.name b/.idea/.name
--- a/.idea/.name	
+++ b/.idea/.name	
@@ -1,1 +1,1 @@
-Hello Zupper
\ No newline at end of file
+PillarsActivity.kt
\ No newline at end of file
